src/ol_infrastructure/applications/mit_open/__main__.py

```diff
@@ -3,37 +3,44 @@
 from pulumi_aws import iam, s3
 from pulumi_vault import aws
 
+from bridge.lib.magic_numbers import FIVE_MINUTES
+
 from ol_infrastructure.lib.aws.iam_helper import IAM_POLICY_VERSION, lint_iam_policy
 from ol_infrastructure.lib.ol_types import AWSBase
 from ol_infrastructure.lib.pulumi_helper import parse_stack
+
 from ol_infrastructure.lib.vault import setup_vault_provider
 
+# Configuration items and initialziations
+if Config("vault_server").get("env_namespace"):
+    setup_vault_provider()
+
 setup_vault_provider()
 mit_open_config = Config("mit_open")
 stack_info = parse_stack()
 aws_config = AWSBase(
     tags={
         "OU": "mit-open",
         "Environment": stack_info.env_suffix,
-        "Application": "mit-open",
+        "Application": "mitopen",
     }
 )
 app_env_suffix = {"ci": "ci", "qa": "rc", "production": "production"}[
     stack_info.env_suffix
 ]
 
-app_storage_bucket_name = f"mit-open-app-storage-{app_env_suffix}"
+app_storage_bucket_name = f"ol-mitopen-app-storage-{app_env_suffix}"
 application_storage_bucket = s3.Bucket(
-    f"mit_open_learning_application_storage_bucket_{stack_info.env_suffix}",
+    f"ol_mitopen_app_storage_bucket_{stack_info.env_suffix}",
     bucket=app_storage_bucket_name,
     versioning=s3.BucketVersioningArgs(enabled=True),
     tags=aws_config.tags,
 )
 
-FIVE_MINUTES = 300
+course_data_bucket_name = f"ol-mitopen-course-data-{app_env_suffix}"
```

Comment:
I'm not certain, but it seems like this corresponds to [this](https://github.com/mitodl/salt-ops/pull/1572/files#diff-41a38d693d2e4da71673c6cdedce7ce28855dffb09911bf4898e37e4da74987dR136)?